content/guides/getting-started/developer-account.md

```diff
@@ -0,0 +1,49 @@
+---
+rank: 1
+related_guides:
+- authentication
+- authorization
+- api-calls
+---
+
+# Create a developer account
+
+Create your developer account to start developing applications with Box.
+Once you create an account, you will get a Developer Enterprise, where
+you can:
+
+- access to the admin-facing tools,
```

Comment:
```suggestion
- access admin-facing tools,
```
or `provide access to` if that was the intended meaning :) I would also skip the article but it's just a gut feeling so it's up to you :D